### PR TITLE
Page file updated

### PR DESCRIPTION
**Pull request by Dheeru12503**

### CODE REVIEW

**Comment by mohdnoorain** [comment]
@Dheeru12503 

>###  Where is main section with image and register form 

I'm calling this Main Section


image

File: src/app/section/MainSection.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import Header from "@/app/components/header/Header"
+export default function MainSection() {
+  return (
+     <Header/>
```
Comment (mohdnoorain) [inline src/app/section/MainSection.tsx]: @Dheeru12503 
Add main section code here.

**Comment by mohdnoorain** [comment]
@Dheeru12503 
add these lines in gitignore file at bottom
```
#package-lock
package-lock.json
```

thene delete this file.

**Comment by mohdnoorain** [comment]
> Also need to improve the css a little here , see the difference

not talking about background

{B262F279-1FD0-4817-9B71-E0538E841B39}

----
{BD4DEBBA-0610-4734-A479-6450EECC85AD}


- Font family 
- font size 
- spacing
- image size

File: src/app/components/header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          </div>
+        </div>
+      )}
+
+      {showSignIn && <LoginForm />}
+    </nav>
+  );
+};
+
+export default NavBar;
```
Comment (mohdnoorain) [inline src/app/components/header/Header.tsx]: @Dheeru12503 
change component name to Header.

File: src/app/section/MainSection.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import Header from "@/app/components/header/Header"
+export default function MainSection() {
```
Comment (mohdnoorain) [inline src/app/section/MainSection.tsx]: @Dheeru12503 
don't export directly 
make it the same as in header
```
const ComponentName: React.FC = () => {
}

export default ComponentName
```

**Comment by Dheeru12503** [comment]
> @Dheeru12503
> 
> > ### Where is main section with image and register form
> 
> I'm calling this Main Section
> 
> image

Done bro

**Comment by Dheeru12503** [comment]
> > Also need to improve the css a little here , see the difference
> 
> not talking about background
> 
> {B262F279-1FD0-4817-9B71-E0538E841B39}
> 
> {BD4DEBBA-0610-4734-A479-6450EECC85AD}
> 
> * Font family
> * font size
> * spacing
> * image size

which font family i should to use ?

**Comment by mohdnoorain** [comment]
> which font family i should to use ? 
> 


Same as in main website.

**Comment by Dheeru12503** [comment]
> > Also need to improve the css a little here , see the difference
> 
> not talking about background
> 
> {B262F279-1FD0-4817-9B71-E0538E841B39}
> 
> {BD4DEBBA-0610-4734-A479-6450EECC85AD}
> 
> * Font family
> * font size
> * spacing
> * image size

Done styling is updated

File: src/app/section/MainSection.tsx (hunk 1 above)
Comment (mohdnoorain) [reply src/app/section/MainSection.tsx]: @Dheeru12503 
is this done ?

File: src/app/section/MainSection.tsx (hunk 3 above)
Comment (mohdnoorain) [reply src/app/section/MainSection.tsx]: @Dheeru12503 
and this ?

**Review by mohdnoorain — CHANGES_REQUESTED** [review]

File: src/app/section/MainSection.tsx (hunk 1 above)
Comment (Dheeru12503) [reply src/app/section/MainSection.tsx]: Done

File: src/app/section/MainSection.tsx (hunk 3 above)
Comment (Dheeru12503) [reply src/app/section/MainSection.tsx]: done

File: src/app/components/heroSection/HeroSection.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 "use client";
 import React, { useEffect, useState } from "react";
 import styles from "./HeroSection.module.css";
-import SignUpForm from "./SignUpForm";
+import SignUpForm from "../signUpForm/SignUpForm";
 
 // import { SignupForm } from "./SignUpForm";
```
Comment (mohdnoorain) [inline src/app/components/heroSection/HeroSection.tsx]: @Dheeru12503 
Add max width in header and main section

File: src/app/components/signUpForm/SignUpForm.tsx
Comment (mohdnoorain) [inline src/app/components/signUpForm/SignUpForm.tsx:1]: @Dheeru12503 
change component name to EnquiryForm

Comment (mohdnoorain) [reply src/app/components/signUpForm/SignUpForm.tsx:1]: add country dropdown

**Review by mohdnoorain — CHANGES_REQUESTED** [review]

File: src/app/components/heroSection/HeroSection.tsx (hunk 4 above)
Comment (Dheeru12503) [reply src/app/components/heroSection/HeroSection.tsx]: Done

File: src/app/components/signUpForm/SignUpForm.tsx
Comment (Dheeru12503) [reply src/app/components/signUpForm/SignUpForm.tsx:1]: done

File: src/app/layout.tsx [hunk 5]
```diff
@@ -26,9 +26,9 @@ export default function RootLayout({
   return (
     <html lang="en">
       <link
```
Comment (mohdnoorain) [inline src/app/layout.tsx]: @Dheeru12503 
wrap this link in head tag.
It's giving error on consoles

**Review by mohdnoorain — CHANGES_REQUESTED** [review]

**Review by mohdnoorain — APPROVED** [review]

**Pull request merged**